Makefile

```diff
@@ -156,6 +160,16 @@ build/test.data: $(CPYTHONLIB) $(UGLIFYJS)
 	)
 	$(UGLIFYJS) build/test.js -o build/test.js
 
+build/distutils.data: $(CPYTHONLIB) $(UGLIFYJS)
+	( \
+		cd $(CPYTHONLIB)/distutils; \
+		find . -type d -name __pycache__ -prune -exec rm -rf {} \; \
+	)
+	( \
+		cd build; \
+		python $(FILEPACKAGER) distutils.data --lz4 --preload ../$(CPYTHONLIB)/distutils@/lib/python$(PYMINOR)/distutils --js-output=distutils.js --export-name=pyodide._module --exclude __pycache__ \
+	)
+	$(UGLIFYJS) build/distutils.js -o build/distutils.js
```

Comment:
Not particularly happy about having to repeat this for each unvendored package, but I'm not sure what else we can do.

I considered creating `meta.yaml` for such stdlib package and treating them as regular package. But that doesn't work as we need to be sure that CPython build artifacts are available (which is not part of the current build / dependency resolution setup)